---
layout: page
permalink: /research/
title: "Research"
modified: 2014-08-03 22:14
tags: [trichoptera, phylogeny, model selection, phylogenetics, machine learning, entomology, insects]
image:
  feature: Rheinbach_woods_banner.jpg
  credit: Paul Frandsen
  creditlink: 
share: 
---

-----

I am interested in a wide variety of research topics, all falling under the broad umbrella of evolutionary biology. When I say this, I mean that I'm interested in how the world's current diversity of life has arisen and what life on earth might have been like in the past. I want to know what organisms looked like, where they lived, and how historical events shaped their distributions. I am fascinated by the variation in organisms' genomes and how these variations might help us better understand their past. To answer some of these questions, I use the tools of molecular phylogenetics and bioinformatics.

###Insect evolution

The world is a wonderful place full of interesting life. Among the most diverse and least understood organisms are the insectsâ€”a fact that makes them endlessly interesting to me. I satisfy some of my curiosity by participating in large collaborative efforts such as the [1KITE](http://1kite.org) project, which is aimed at better understanding the evolutionary history and diversity of insects. My specific insect interests are focused on an order of small, moth-like insects called caddisflies (Insecta: Trichoptera), and I am spending a good portion of my PhD in an attempt to better understand the evolution this group. You can read more about my research on caddisflies <a markdown="0" href="{{ site.url }}/trichoptera/">here</a>. 

###Phylogenetic methods

A phylogenetic tree is only as good as the method and data used to infer it. I am interested in improving phylogenetic tree estimation by ensuring that we are using appropriate models of molecular evolution. I am working with great collaborators like [Rob Lanfear](http://robertlanfear.com) and [Brett Calcott](http://www.brettcalcott.com) to devise data-driven methods to partition and apply models of molecular evolution to DNA and protein sequence alignments. We are implementing these methods in the software [PartitionFinder](http://www.robertlanfear.com/partitionfinder/). In compliment, I am interested in targeting those data best suited for these models. To do this, I am working on developing additional software that harnesses the power of various machine learning methods to identify and remove portions of DNA sequence alignments that are in violation of model assumptions.
